### ISSUE TITLE
Visual AND auditory stimuli in the same trial

### ISSUE THREAD

**Issue by Miko999** [jspsych/jsPsych#2607]
Hello!

I'm quite new to javascript and jspsych and am attempting to implement a metronome response task (see: https://doi.org/10.3758/s13414-020-02131-x), mostly using the audio-keyboard-response plugin.
This is a continuous performance task with regularly spaced audio stimuli (i.e., metronome beats), where participants have to tap a key along with the stimuli. The stimuli must be equally spaced to be 'rhythmic' for the purposes of this task.
The original had visual stimuli appear when participants responded, depending on the timing of their response relative to the auditory stimuli. I wanted to do something similar to show that the participants' response has been recorded. I could add this in using "on_finish", but I'm not sure if I can tie it to the response within the same trial. Along with that, I wanted to have a fixation cross on screen at all times (though it could be covered by the visual stimuli from the response).

So basically:
**- Can I get a visual stimulus to appear during an audio-keyboard-response trial when the participant responds, regardless of when they respond during the trial? (If so, how?)
- Can I have a background image in audio-keyboard-response trials, to include a fixation cross? (If so, how?)

Any advice would be much appreciated!**
I have to do this all in Gorilla(.sc), since that's how my supervisor wants us to implement online tasks, and jspsych seems like the best option (the Gorilla 'task creator' GUI has very limited logic/branching options).
